docusaurus/docs/iOS/uikit/components/channel-list.md

```diff
@@ -111,6 +111,56 @@ class ViewController: ChatChannelListVC {
 }
 ```
 
+### Loading states
+
+You can opt to show an empty, error and loading view by setting the following flag to true in the `Components` config:
+
+```swift
+Components.isChatChannelListDefaultLoadingStatesEnabled = true
+```
+
+This feature is disabled by default. By setting this feature on, you can use show views for each state with no extra code.
+
+| Empty | Error | Loading |
+| ------------- | ------------- | ------------- |
+| ![Empty View](../../assets/channel-list-empty-view.png) | ![Error View](../../assets/channel-list-error-view.png) | ![Loading View](../../assets/channel-list-loading-view.png) |
+
+You can further customize or add your own implementation of an empty, error or loading view by subclassing `ChatChannelListEmptyView`, `ChatChannelListErrorView` or `ChatChannelListLoadingView` respectively, add your custom views as properties and overriding our custom life cycle methods `setUp()`, `setUpAppearance()`, `setUpLayout()` and `updateContent()` to configure your custom views.
```

Comment:
```suggestion
You can further customize or add your own implementation of an empty, error or loading view by subclassing `ChatChannelListEmptyView`, `ChatChannelListErrorView` or `ChatChannelListLoadingView` respectively.
```
No need to add additional information, at this point in the documentation the customer should be already familiar with our custom lifecycles